src/ontology/cl-edit.owl

```diff
@@ -31920,6 +31921,19 @@ AnnotationAssertion(rdfs:label obo:CL_4040005 "mesenchymal stem cell of apical p
 SubClassOf(obo:CL_4040005 obo:CL_0000134)
 SubClassOf(obo:CL_4040005 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003677))
 
+# Class: obo:CL_4042002 (TAC3-positive medium spiny neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523") obo:IAO_0000115 obo:CL_4042002 "A nucleus accumbens shell and olfactory tubercle D1 medium spiny neuron that expresses TAC3. This medium spiny neuron has an enriched expression of the genes MPPED1, HPCAL1, MEIS3. ")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042002 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042002 "2024-03-05T13:36:54Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523") oboInOwl:hasExactSynonym obo:CL_4042002 "D1Sh TAC3 archetype")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasExactSynonym obo:CL_4042002 "TAC3-positive D1Sh-MSN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasExactSynonym obo:CL_4042002 "TAC3-positive MSN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523") rdfs:comment obo:CL_4042002 "In Rhesus Macaque, TC3-positive medium spiny neuron has its soma in a cluster in the medial shell region of the nucleus accumbes and co-expresses the DRD1 receptor.")
```

Comment:
TC3 --> TAC3
This cell is classified as a 'nucleus accumbens shell and olfactory tubercle D1 medium spiny neuron', which expresses DRD1. Therefore, I think "and co-expresses the DRD1 receptor" is needed.

Reply:
Is the location of the cell type also variable? If not, I don't think it is necessary the first In Rhesus Macaques, as we use it more for sets of genes that are species specific and we are unsure they are in other species. Also, I would remove the last bit of the sentence since it is in the textual definition and it's a characteristic of the parent class:
> A TAC3-positive medium spiny neuron has its soma in a cluster in the medial shell region of the nucleus accumbens. In Rhesus Macaque, this medium spiny neuron has an enriched expression of the genes MPPED1, HPCAL1, MEIS3.

Accumbens is missing an 'n'. The first sentence could be the gloss of the textual definition if it's common in all species.